examples/SimpleProducer/Program.cs

```diff
@@ -31,9 +31,13 @@ public static void Main(string[] args)
             string brokerList = args[0];
             string topicName = args[1];
 
-            var config = new Dictionary<string, object> { { "bootstrap.servers", brokerList } };
+            var config = new Dictionary<string, object> 
+            { 
+                { "bootstrap.servers", brokerList },
+                { "string.serializer.encoding.key", "UTF8" }
+            };
 
-            using (var producer = new Producer<Null, string>(config, null, new StringSerializer(Encoding.UTF8)))
+            using (var producer = new Producer<Null, string>(config, null, new StringSerializer(config)))
```

Comment:
An example of use. You'd only configure the StringSerializer via the config object like this if you wanted to allow it to be configurable via the same config file used for the producer. This provides an easy mechanism for that. Otherwise, you'd just pass in the encoding object directly.